fnl/paperplanes/providers/srht.fnl

```diff
@@ -0,0 +1,34 @@
+(local {: set-field : opts-to-fields} (require :paperplanes.util.providers))
+(local fmt string.format)
+
+(fn make [content-arg meta opts]
+  (local args (doto []
+                    (table.insert :--header)
+                    (table.insert (fmt
+                                    "Authorization:'token %s'"
```

Comment:
@p00f Make this "Authorization: token %s", `uv.spawn`/exec will automatically wrap it.

Reply:
Also apply similar to the content type header.

Reply:
right, i needed to do that to the json payload too